app/presenters/vm_presenter.rb

```diff
@@ -0,0 +1,12 @@
+class VmPresenter
+  def initialize(controller, session)
+    @controller = controller
```

Comment:
Having been reduced to "get the thing from the DB object", this is no longer feeling like the place to have a presenter discussion, to me. Let's just inline the `BinaryBlob.find` back into the callers.